tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -162,6 +162,22 @@ module.exports = {
       for (const btn of cancelButtons) {
         await this.api.elementIdClick(btn.ELEMENT).waitForAnimationToFinish()
       }
+    },
+    browseToUserProfile: async function () {
+      return this.click('@userProfile')
+    },
+    getUserProfileElements: async function () {
```

Comment:
Can we somehow only extract username here and rename the function to `getUserName` or something similar?

Reply:
we might need all the elements for next scenario related to this feature

Reply:
Can we not only get the text from wherever the element for username is? I think you are overengineering it. If in the future, any texts get changed on the container, all the consumers of this function needs to be adjusted (besides revealing all the text of this container, which is wrong, anyway).

Reply:
done